advanced_samples/CSV/dotnet/CSVtoOCS_pkce/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
```

Comment:
I think you can delete the IIS stuff in here unless you think we should keep it. I'm using something like this:

```json
{
  "profiles": {
    "AuthorizationCodeFlow": {
      "commandName": "Project",
      "launchBrowser": false,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}
```